boards.txt

```diff
@@ -19913,6 +19911,8 @@ m5stack_atom.menu.EraseFlash.all.upload.erase_cmd=-e
 ##############################################################
 
 m5stack_atoms3.name=M5AtomS3
+m5stack_atoms3.vid.0=0x303a
+m5stack_atoms3.pid.0=0x1001
```

Comment:
these are non-unique PID and VID. Please remove them from all boards!

Reply:
@P-R-O-C-H-Y please make sure that such things do not pass!!!

Reply:
yep, missed that.

Reply:
But I see that many boards have added it. Is it enough to just use a different PID and VID?

<img width="359" alt="image" src="https://github.com/user-attachments/assets/8de3365f-0d47-44bc-8239-2793cfa5618e">

Reply:
the boards that use different PID and VID, use it through the OTG port (TinyUSB). The ones you had are for the integrated CDC/JTAG port and should only be used/added to the "ESP32 Family Board" Else there are issues with the Arduino IDE and detecting boards.

Reply:
Thank you for your reply.